index.bs

```diff
@@ -2789,6 +2789,31 @@ the [=authenticator=] MUST:
     attStmtTemplate .within $$attStmtType
     ```
 
+### Signature Formats for Packed Attestation, FIDO U2F Attestation, and Assertion Signatures ### {#signature-attestation-types}
+    - For COSEAlgorithmIdentifier -7 (ES256),  and other ECDSA-based algorithms,
+        a signature is encoded as an ASN.1 DER Ecdsa-Sig-Value, as defined in [[RFC3279]] section 3.3.1.
+
+        Note: As CTAP1/U2F devices are already producing signatures in this format, CTAP2
+        devices will also produce signatures in same format, for consistency reasons.
+        It is recommended that any new attestation formats defined not use ASN.1 encodings,
+        but instead represent signatures as equivalent fixed-length byte arrays without internal structure,
+        using the same representations as used by COSE signatures as defined in [[!RFC8152]].
+
+        ```
```

Comment:
it appears that the below example is ASN.1-wrapped. If so, I would move it up above the Note: (presently above), because having it down here is confusing -- i was expecting a "raw" fixed-length example to go with the Note.

Reply:
Moved.